content/v2.0/actions/parameters.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Parameters
+order: 30
+---
+
+Parameters are taken from the Rack env, wrapped into the `Params` object, and passed as an argument to `#handle` method.
+
+They are similar to a Ruby `Hash`, but they offer an expanded set of features.
+
+## Sources
+
+Params can come from:
+
+  * [Router variables](/v2.0/router/basic-usage) (eg. `/books/:id`)
+  * Query string (eg. `/books?title=Hanami`)
+  * Request body (eg. a `POST` request to `/books`)
+
+## Access
+
+To access the value of a param, we can use the _subscriber operator_ `#[]`.
+
+```ruby
+# app/controllers/dashboard/index.rb
+
+module Bookshelf
+  module Actions
+    module Books
+      class Index < Action
+        def handle(req, res)
```

Comment:
We're using `request`, and `response` after all so it should be used here as well.

Reply:
Addressed.